provisioning/service/src/Auth/SharedAccessSignature.cs

```diff
@@ -33,51 +29,23 @@ private SharedAccessSignature(string shareAccessSignatureName, DateTime expiresO
                 throw new UnauthorizedAccessException("The specified SAS token is expired");
             }
 
-            _shareAccessSignatureName = shareAccessSignatureName;
-            _signature = signature;
-            _audience = WebUtility.UrlDecode(encodedAudience);
+            ShareAccessSignatureName = shareAccessSignatureName;
+            Signature = signature;
+            Audience = WebUtility.UrlDecode(encodedAudience);
             _encodedAudience = encodedAudience;
             _expiry = expiry;
-            _keyName = keyName ?? string.Empty;
+            KeyName = keyName ?? string.Empty;
         }
 
-        public string ShareAccessSignatureName
-        {
-            get
-            {
-                return _shareAccessSignatureName;
-            }
-        }
+        public string ShareAccessSignatureName { get; private set; }
```

Comment:
Is this per standard? public fields below constructor?

Reply:
yes, since this is a public property, it would be placed below any constructors